pkg/kubehound/graph/edge/pod_patch_namespace.go

```diff
@@ -35,13 +35,24 @@ func (e *PodPatchNamespace) Name() string {
 	return "PodPatchNamespace"
 }
 
+func (e *PodPatchNamespace) AttckTechniqueID() AttckTechniqueID {
+	return AttckTechniqueCreateOrModifySystemProcessContainerService
+}
+
+func (e *PodPatchNamespace) AttckTacticID() AttckTacticID {
+	return AttckTacticPersistence
+}
+
```

Comment:
This does not match the `docs/reference/attacks/POD_PATCH.md`, but I prefer the IDs you used here in this specific case.
I think we can let it that way, the only downside could be that it doesn't appear in the doc

Reply:
Fixed in code.